README.md

```diff
@@ -6,13 +6,38 @@ The SBOM tool is a highly scalable and enterprise ready tool to create SPDX 2.2
 
 ## Table of Contents
 
-* [Installation](docs/installation.md)
-* [Building and running Sbom tool](docs/build-and-run.md)
+* [Installation](#installation)
+* [Run the tool]()
+* [Building and running Sbom tool from source](docs/build-and-run.md)
 * [Telemetry](#Telemetry)
 * [Contributing](#Contributing)
 * [Security](#Security)
 * [Trademarks](#Trademarks)
 
+## Installation
+
+### Windows, Mac and Linux executable.
+Please check the [Releases](https://github.com/microsoft/sbom-tool/releases) page to go to the version of the tool you want to install. Then download the tool from the release assets for the required runtime. 
+
+Please check the [arguments](docs/sbom-tool-arguments.md) that you can provide to the sbom tool.
+
+### Sbom tool C# Api
```

Comment:
Capitalize the SBOM

Reply:
Fixed

Reply:
"Sbom" is still not capitalized on my end.